src/services/projects.js

```diff
@@ -529,4 +625,46 @@ module.exports = class ProjectsHelper {
 			return error
 		}
 	}
+
+	/**
+	 * Upload to cloud
+	 * @method
+	 * @name uploadToCloud
+	 * @param {Object} req.id - project id
+	 * @returns {JSON} - upload project response.
+	 */
+
+	static async uploadToCloud(fileName, resourceId, loggedInUserId, bodyData) {
```

Comment:
@priyanka-TL Move this to a common module will be used for other resources as well, this function can be a generic function and has no specific mention of project